modules/client-configuration/pages/registration-methods-saltbundle.adoc

```diff
@@ -0,0 +1,32 @@
+[[registration-methods-saltbundle]]
+= Client Registration with Salt Bundle
+
+Since end of 2021, the registration method with the so-called Salt bundle is available.
```

Comment:
Do we need to mention the date? I am also not sure "the so-called Salt bundle" is very official :-)

Reply:
```suggestion
The registration method with the Salt bundle is the recommended registration method as long as it does not cover your use case.
This section explains the advantages and limitations of the current implementation.
```